osquery/tables/system/tests/BUCK

```diff
@@ -75,96 +28,24 @@ osquery_cxx_test(
         osquery_target("osquery/core/sql:core_sql"),
         osquery_target("osquery/database:database"),
         osquery_target("osquery/filesystem:osquery_filesystem"),
-        osquery_target("osquery/remote/tests:remote_test_utils"),
         osquery_target("osquery/tables/system:system_table"),
-        osquery_target("osquery/utils:utils"),
-        osquery_target("osquery/utils/conversions:conversions"),
-        osquery_target("plugins/database:ephemeral"),
-    ],
-)
-
-osquery_cxx_test(
-    name = "known_hosts_tests",
-    platform_srcs = [
-        (
-            POSIX,
-            [
-                "posix/known_hosts_tests.cpp",
-            ],
-        ),
-    ],
-    visibility = ["PUBLIC"],
-    deps = [
-        osquery_target("osquery/config/tests:test_utils"),
-        osquery_target("osquery/core:core"),
-        osquery_target("osquery/core/sql:core_sql"),
-        osquery_target("osquery/database:database"),
-        osquery_target("osquery/filesystem:osquery_filesystem"),
         osquery_target("osquery/remote/tests:remote_test_utils"),
-        osquery_target("osquery/tables/system:system_table"),
-        osquery_target("osquery/tables/system:system_table"),
         osquery_target("osquery/utils:utils"),
         osquery_target("osquery/utils/conversions:conversions"),
         osquery_target("plugins/database:ephemeral"),
     ],
 )
 
 osquery_cxx_test(
-    name = "shell_history_tests",
+    name = "posix_tests",
     platform_srcs = [
         (
             POSIX,
             [
+                "posix/apt_sources_tests.cpp",
```

Comment:
I can make this change in the PR, but we do crazy things like install `yum` and `rpm` on macOS. I wanted to support as many use cases as possible since it was trivial (no dependencies needed in this case).

Reply:
Oh ok then, I mean I knew that the (non-existing) package manager wasn't that good, but I didn't know you could resort to that! :D